app/screens/more_dms/index.js

```diff
@@ -27,6 +30,7 @@ function mapStateToProps(state) {
         theme: getTheme(state),
         currentDisplayName: state.views.channel.displayName,
         currentUserId: getCurrentUserId(state),
+        isGuest: currentUser && isGuest(currentUser),
```

Comment:
when would the current user not be defined? I'm ok as it is,.. just wanted to understand

Reply:
Yes, fair point, it shouldn't be undefined.

Reply:
changed